### PR TITLE
Feat/pci ai notebooks manage notebook pop up (Start, Stop, Delete)

### PR DESCRIPTION
**Pull request by abullet33**
ref:DATATR-1559,DATATR-1560,DATATR-1561

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStartNotebook.hook.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    mutationFn: (notebookInfo: NotebookData) => {
+      return startNotebook(notebookInfo);
+    },
+    onError,
+    onSuccess: () => {
+      onStartSuccess();
+      // Invalidate service list query to get the latest data
+      queryClient.invalidateQueries({
+        queryKey: [projectId, 'ai/notebook'],
+      });
```
Comment (jperchoc) [inline packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStartNotebook.hook.tsx:27]: Should we also invalidate this query ?   const queryKey = [projectId, 'ai/notebook', notebookId];

File: packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStopNotebook.hook.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    mutationFn: (notebookInfo: NotebookData) => {
+      return stopNotebook(notebookInfo);
+    },
+    onError,
+    onSuccess: () => {
+      onStopSuccess();
+      // Invalidate service list query to get the latest data
+      queryClient.invalidateQueries({
+        queryKey: [projectId, 'ai/notebook'],
+      });
```
Comment (jperchoc) [inline packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStopNotebook.hook.tsx:24]: Same here, maybe we need to invalidate the getNotebook query

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx
Comment (jperchoc) [inline packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx:1]: It seems that there is temporary code in this file, can you remove it ?

**Review by jperchoc — APPROVED** [review]

File: packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStartNotebook.hook.tsx (hunk 1 above)
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStartNotebook.hook.tsx:27]: Seen together, invalidateQueries will invalidate all queries starting by ai/notebook

File: packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStopNotebook.hook.tsx (hunk 2 above)
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/useStopNotebook.hook.tsx:24]: Seen together, invalidateQueries will invalidate all queries starting by ai/notebook

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx:1]: Fixed

**Pull request merged**